standalone/content/js/standaloneRoomViews.jsx

```diff
@@ -783,6 +802,7 @@ loop.standaloneRoomViews = (function(mozL10n) {
     },
 
     render: function() {
+      var screenSharingPaused = this.props.activeRoomStore.getStoreState("streamPaused");
```

Comment:
The activeRoomStore state is already included in the view - see `getInitialState`/`componentWillMount` - all you need to do here is `this.state.streamPaused` (and you can drop the intermediate variable.